CRM/MembershipExtras/Test/Fabricator/FinancialAccount.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+use CRM_MembershipExtras_Test_Fabricator_Base as BaseFabricator;
+
+class CRM_MembershipExtras_Test_Fabricator_FinancialAccount extends BaseFabricator {
+
+  protected static $entityName = 'FinancialAccount';
+
+  protected static $defaultParams = [
```

Comment:
these are not defaults, these are very specific values for a very specific scenario, I would rather keep them empty and set their values in the test classes that use this fabricator, same in EntityFinancialAccount